fuse/fuse.h

```diff
@@ -0,0 +1,30 @@
+/* Wake FUSE launcher to capture inputs/outputs
+ *
+ * Copyright 2019 SiFive, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You should have received a copy of LICENSE.Apache2 along with
+ * this software. If not, you may obtain a copy at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FUSE_H
+#define FUSE_H
+
+#include <string>
+
+int run_fuse(
+	const std::string& daemon_path,
+	const std::string& working_dir,
+	const std::string& json,
+	std::string& result_json);
```

Comment:
This does more than just run fuse; it also spawn the subprocess, `run_IN_fuse`?
It might make sense down the line to split this function up a bit more.